src/main/java/net/smartcosmos/dto/metadata/MetadataResponse.java

```diff
@@ -19,26 +19,26 @@
     @Setter(AccessLevel.NONE)
     private int version = VERSION; // just in case there is a default constructor sometime
 
+    private final String id;
+    private final String ownerType;
+    private final String ownerId;
     private final String dataType;
-    private final String entityReferenceType;
-    private final String key;
-    private final Long lastModifiedTimestamp;
-    private final String rawValue;
-    private final String referenceUrn;
-    private final String urn;
-    private final String moniker;
+    private final String keyName;
+    private final String value;
+    private final String tenantId;
 
     @Builder
-    @ConstructorProperties({"dataType", "entityReferenceType", "key", "lastModifiedTimestamp", "rawValue", "referenceUrn", "urn", "moniker"})
-    public MetadataResponse(String dataType, String entityReferenceType, String key, Long lastModifiedTimestamp, String rawValue, String referenceUrn, String urn, String moniker) {
+    @ConstructorProperties({"id", "ownerType", "ownerId", "dataType", "keyName", "value", "tenantId"})
+    public MetadataResponse(
```

Comment:
This response would require processing before returning as a REST response, because this will look like this:
```
{
"value": "some arbitrary JSON Object"
}
```

Therefore, we probably don't need a `MetadataResponse` object (for now). Returning `Map<String, Object>` might be sufficient.